rllib/agents/pg/pg.py

```diff
@@ -33,6 +36,24 @@
 # yapf: enable
 
 
+def validate_config(config: TrainerConfigDict) -> None:
```

Comment:
Switch on by default for PG.

Reply:
Please set this in the algorithm config instead of in the code, here and elsewhere.